lib/util/commandSplit.js

```diff
@@ -0,0 +1,37 @@
+const conf = require(__dirname + '/conf');
+
+function commandSplit(com) {
+  let command, wikiaDomain, wikiaSearch;
```

Comment:
My only true concern with this PR is util/commandSplit is very specific to wikia. Which is fine - but name the file as such.